packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsPending/TransactionsPending.tsx

```diff
@@ -13,16 +14,26 @@ type TProps = {
 };
 
 const TransactionsPending: React.FC<TProps> = ({ filter = 'all' }) => {
-    const { data, resetData, subscribe, unsubscribe } = useCryptoTransactions();
+    const { data: transactions, isLoading, isSubscribed, resetData, subscribe, unsubscribe } = useCryptoTransactions();
+    const [shouldShowLoader, setShouldShowLoader] = useState(true);
```

Comment:
Why do we need this state ?
We can just show the loader based on conditions 🤔

Reply:
Have updated this